cmdstanpy/install_cmdstan.py

```diff
@@ -161,68 +129,53 @@ def compile_example() -> None:
             os.path.join('examples', 'bernoulli', 'bernoulli' + EXTENSION)
         ).as_posix(),
     ]
-    proc = subprocess.Popen(
-        cmd,
-        cwd=None,
-        stdin=subprocess.DEVNULL,
-        stdout=subprocess.PIPE,
-        stderr=subprocess.PIPE,
-        env=os.environ,
-    )
-    while proc.poll() is None:
-        if proc.stdout:
-            proc.stdout.readline().decode('utf-8')
-    _, stderr = proc.communicate()
-    if proc.returncode:
-        msgs = ['Failed to compile example model bernoulli.stan']
-        if stderr:
-            msgs.append(stderr.decode('utf-8').strip())
-        raise CmdStanInstallError('\n'.join(msgs))
-
-
-def rebuild_cmdstan(verbose: bool = True) -> None:
+    try:
+        do_command(cmd)
+    except RuntimeError as e:
+        # pylint: disable=raise-missing-from
+        raise CmdStanInstallError(f'Command "make clean-all" failed\n{e}')
+
+
+def rebuild_cmdstan() -> None:
     """
     Rebuilds the existing CmdStan installation.
     This assumes CmdStan has already been installed,
     though it need not be installed via CmdStanPy for
     this function to work.
-
-    :param verbose:  Boolean value; when ``True``, output from CmdStan build
-        processes will be streamed to the console.  Default is ``False``.
     """
     try:
         with pushd(cmdstan_path()):
-            clean_all(verbose)
-            build(verbose)
+            clean_all()
+            build()
             compile_example()
     except ValueError as e:
         raise CmdStanInstallError(
             "Failed to rebuild CmdStan. Are you sure it is installed?"
         ) from e
 
 
-def install_version(
-    cmdstan_version: str, overwrite: bool = False, verbose: bool = False
-) -> None:
+def install_version(cmdstan_version: str, overwrite: bool = False) -> None:
```

Comment:
So is verbose gone or is verbose on by default? I do not think it should be on by default

Reply:
I just reverted everything back to the way it was - verbose is off by default.